tests/components/config/test_core.py

```diff
@@ -4,35 +4,96 @@
 
 from homeassistant.bootstrap import async_setup_component
 from homeassistant.components import config
+from homeassistant.components.websocket_api.const import TYPE_RESULT
 from tests.common import mock_coro
 
 
-@asyncio.coroutine
-def test_validate_config_ok(hass, hass_client):
+async def test_validate_config_ok(hass, hass_client):
     """Test checking config."""
     with patch.object(config, 'SECTIONS', ['core']):
-        yield from async_setup_component(hass, 'config', {})
+        await async_setup_component(hass, 'config', {})
 
-    yield from asyncio.sleep(0.1, loop=hass.loop)
+    await asyncio.sleep(0.1, loop=hass.loop)
 
-    client = yield from hass_client()
+    client = await hass_client()
 
     with patch(
         'homeassistant.components.config.core.async_check_ha_config_file',
             return_value=mock_coro()):
-        resp = yield from client.post('/api/config/core/check_config')
+        resp = await client.post('/api/config/core/check_config')
 
     assert resp.status == 200
-    result = yield from resp.json()
+    result = await resp.json()
     assert result['result'] == 'valid'
     assert result['errors'] is None
 
     with patch(
         'homeassistant.components.config.core.async_check_ha_config_file',
             return_value=mock_coro('beer')):
-        resp = yield from client.post('/api/config/core/check_config')
+        resp = await client.post('/api/config/core/check_config')
 
     assert resp.status == 200
-    result = yield from resp.json()
+    result = await resp.json()
     assert result['result'] == 'invalid'
     assert result['errors'] == 'beer'
+
+
+async def test_websocket_core_update(hass, hass_ws_client):
+    """Test core config update websocket command."""
+    with patch.object(config, 'SECTIONS', ['core']):
+        await async_setup_component(hass, 'config', {})
+
+    client = await hass_ws_client(hass)
+    await client.send_json({
+        'id': 5,
+        'type': 'config/core/update',
+        'latitude': 123,
+    })
+
+    msg = await client.receive_json()
+
+    assert msg['id'] == 5
+    assert msg['type'] == TYPE_RESULT
+    assert msg['success']
```

Comment:
Please verify latitude got set, preferably test all other values too

Reply:
Yes, but I had a review comment which I forgot to publish: should the new WS command `config/core/update` return:
- The set value(s)
- The updated core config

Or is it fine like it is?